framework/PageForm/PageForm.tsx

```diff
@@ -17,16 +16,26 @@ import {
   Path,
   useForm,
   UseFormReturn,
-  useFormState,
 } from 'react-hook-form';
-import { useTranslation } from 'react-i18next';
 import { Scrollable } from '../components/Scrollable';
 import { useBreakpoint } from '../components/useBreakPoint';
 import { SettingsContext } from '../Settings';
 import { useFrameworkTranslations } from '../useFrameworkTranslations';
 import { ErrorAlert } from './ErrorAlert';
 import { genericErrorAdapter } from './genericErrorAdapter';
 import { ErrorAdapter } from './typesErrorAdapter';
+import { PageFormSubmitButton, PageFormCancelButton } from './PageFormButtons';
+import styled from 'styled-components';
+
+const FormContainer = styled(PageSection)`
```

Comment:
why is the form body using m-action--MarginTop?

Reply:
switched to a more appropriate variable